_i18n/en.yml

```diff
@@ -501,6 +501,11 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qasicresistance: "What is ASIC resistance? Why is it important?"
```

Comment:
```suggestion
  qasicresistance: "What is an ASIC-resistant mining algorithm? Why is it important?"
```

Reply:
This suggestion slightly changes the meaning of the entry and adds unnecessary complexity. I prefer to keep the original question.